### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by dcmcshan** [NolanKingdon/MMM-MoonPhase#35]
Hi there.  Really enjoying your module.  Wondering if you can add a feature to publish the moon phase like weather does so I can use compliments based on the phase of the moon.  Thank you!

**Comment by NolanKingdon** [comment]
Hey - cool idea.

I can add the code to publish the moonphase in this module, but in order for for the moonphase to be picked up by the compliments module, we need to get some changes in to that module as well.

I opened a feature request in the magic mirror core repository to add this feature. If the core magic mirror team also thinks moon phase compliments is cool, I can make the changes in both spots to make this happen.

I'll keep you up to date with how that goes

**Comment by NolanKingdon** [comment]
I added in the code to emit the notification, but wasn't able to get the code to receive the notification into the default compliments module.

However, I made [an exact copy of the compliments module](https://github.com/NolanKingdon/MMM-compliments) that adds that functionality. To get moonphase related compliments:
1. [Follow the install instructions for the new compliments module](https://github.com/NolanKingdon/MMM-compliments#install)
2. Replace `module: "compliments"` with `module: "MMM-compliments"` in your `config.js`
3. [Follow the instructions for adding third party compliments](https://github.com/NolanKingdon/MMM-compliments#example-use-with-third-party-payload) to your `config.js` using [the moonphases listed in this module's README](https://github.com/NolanKingdon/MMM-MoonPhase#compliments-integration)
